cms/templates/index.html

```diff
@@ -495,6 +523,74 @@ <h4 class="title">${_('Your Course Creator Request Status:')}</h4>
       </div>
       % endif
 
+
+      %if len(courses) > 0 or optimization_enabled:
+      <div class="archive archive-tab">
+        <ul class="list-courses">
+          %for course_info in sorted(courses, key=lambda s: s['display_name'].lower() if s['display_name'] is not None else ''):
+          % if course_info['archived'] == True:
+            <li class="course-item" data-course-key="${course_info['course_key']}">
+              <a class="course-link" href="${course_info['url']}">
+                <h3 class="course-title">${course_info['display_name']}</h3>
+
+                <div class="course-metadata">
+                  <span class="course-org metadata-item">
+                    <span class="label">${_("Organization:")}</span> <span class="value">${course_info['org']}</span>
+                  </span>
+                  <span class="course-num metadata-item">
+                    <span class="label">${_("Course Number:")}</span>
+                    <span class="value">${course_info['number']}</span>
+                  </span>
+                  <span class="course-run metadata-item">
+                    <span class="label">${_("Course Run:")}</span> <span class="value">${course_info['run']}</span>
+                  </span>
+                </div>
+              </a>
+
+              <ul  class="item-actions course-actions">
+                % if allow_course_reruns and rerun_creator_status and course_creator_status=='granted':
```

Comment:
please add spaces around == at the end of the line

Reply:
Sorry, didn't notice about this. I'm borrow this part from course card